scripts/beck/st/error_injection.py

```diff
@@ -147,14 +159,14 @@ def recordOffset(record, i, size):
 
 def ReadTypeSize(byte):					#Ex: 0001(ver) 0009(type) 00003dd8(size) 
 	ver=byte[:4]						#.ot_version = src->hd_bits >> 48,
-	type=byte[6:8]						#.ot_type    = src->hd_bits >> 32 & 0x0000ffff,
+	rtype=byte[6:8]						#.ot_type    = src->hd_bits >> 32 & 0x0000ffff,
```

Comment:
Should this be:
```
rtype=byte[4:8]
```
Why is it [6:8]?

Reply:
No. Dropped a few bit for a specific reason. which no need to mention

Reply:
"No need to mention"?

Please add comments here to mention this.
If the specific reason is not needed to explain, at least to mention:
```
A few bit is dropped for a specific reason.
```
Otherwise, reviewer and later code maintainer will not understand this.